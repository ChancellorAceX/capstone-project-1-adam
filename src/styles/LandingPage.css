header.landingHeader {
  width:100vw;
  display:flex;
  flex-direction: column;
  justify-content: center;
  align-items:center;
}

main.landing {
  position:fixed;
  top:15vh;
  width:100vw;
  height:77vh;
  overflow:auto;
  background-color: black;
  color:#aaaaaa;
  display:flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
}

p.landing {
  width:75vw;
  text-align: center;
  font-size: 20px;
}