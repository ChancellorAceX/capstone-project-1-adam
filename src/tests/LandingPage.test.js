import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import LandingPage from '../components/LandingPage';

describe('LandingPage component', () => {
  it('renders the full login component', () => {
    const wrapper = shallow(<LandingPage />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});