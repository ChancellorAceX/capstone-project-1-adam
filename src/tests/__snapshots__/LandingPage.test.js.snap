// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage component renders the full login component 1`] = `
<div
  className="landing"
>
  <header
    className="landingHeader"
  >
    <h1
      className="landingHeader"
    >
      Welcome to BattleSource!
    </h1>
  </header>
  <main
    className="landing"
  >
    <p
      className="landing"
    >
      This application has two parts.
    </p>
    <p
      className="landing"
    >
      The first is the bestiary, in which you can search the bestiary for monsters based on name,type,and CR. Feel free to click the magnifying glass below to try this out!
    </p>
    <p
      className="landing"
    >
      In the second you can view and manage your own personal campaigns, encounters, and characters. Click the map below to log in! (Clicking the map once logged in will auto-route you to your encounters page)
    </p>
  </main>
</div>
`;
